packages/xo-web/src/xo-app/jobs/schedules/index.js

```diff
@@ -177,13 +181,13 @@ export default class Schedules extends Component {
     this.setState(
       {
         cronPattern: DEFAULT_CRON_PATTERN,
+        enabled: true,
```

Comment:
So we're changing the behaviour?

Reply:
Yes, the schedule will be enabled by default and when we reset it, we need to enable it.